crates/collab/src/tests/test_server.rs

```diff
@@ -271,6 +271,12 @@ impl TestServer {
             node_runtime: FakeNodeRuntime::new(),
         });
 
+        let os_keymap = if cfg!(target_os = "linux") {
+            "keymaps/default-linux.json"
+        } else {
+            "keymaps/default-macos.json" // Default case if none of the above
```

Comment:
I don't think we need this comment here:

```suggestion
            "keymaps/default-macos.json"
```